app/src/main/java/com/aptoide/uploader/apps/view/MyAppsAdapter.java

```diff
@@ -3,26 +3,36 @@
 import android.support.annotation.NonNull;
 import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
+import android.view.View;
 import android.view.ViewGroup;
 import com.aptoide.uploader.R;
 import com.aptoide.uploader.apps.InstalledApp;
+import io.reactivex.Observable;
+import io.reactivex.subjects.PublishSubject;
+import java.util.ArrayList;
 import java.util.List;
 
 public class MyAppsAdapter extends RecyclerView.Adapter<AppViewHolder> {
 
   private final List<InstalledApp> list;
+  private final List<Integer> selectionList;
+  private final MyAppsClickListener listener;
+  private final PublishSubject<Boolean> selectedPublisher;
```

Comment:
naming, variable names should not have its type specified, selection"List" for example. We should change all variables (all but selectedPublisher).

Reply:
Fixed